app/build-js.sh

```diff
@@ -20,11 +20,24 @@ if ! which npm 1>/dev/null 2>&1; then
     exit 1
 fi
 
+# Add navitron & its dependencies to the app bundle
+pushd ../node_modules/navitron
```

Comment:
On other scripts where the project root differs from the script path, I've done exactly what your second option is: introduce `$ROOT` at the top of the script and use that everywhere.